### PR TITLE
Update Redis tutorial

### LINKED ISSUE #1126: Update Redis tutorial
* The Prerequisites says to have the redis bundle installed, but the next section tells user to install the bundle.  Remove one or the other.

* IMO, the note "If the bundle already exists, no action is required." is not necessary.  

* "systemctl start redis" should be "sudo systemctl start redis". 

* "Optional: If you wish to apply the advanced configuration, copy the redis.conf into /etc/ directory." - What does it mean to "apply advanced configuration"?  

* All Redis CLI prompts shouldn't use "bash" for the code-block header.  Use ".. code-block:: none"

* "sudo -E docker pull clearlinux/redis" - I don't think the "-E" is necessary for the tutorial.  I believe this is for passing environment variables.  

* Need to a step for enabling Redis to start on boot.  "sudo systemctl enable redis"

### PR DESCRIPTION
**Pull request by bktan8**
Closes #1126

Signed-off-by: Bun K Tan <bun.k.tan@intel.com>

### CODE REVIEW

File: source/tutorials/redis.rst [hunk 1]
```diff
@@ -15,74 +15,65 @@ infrastructure development.
...
-  Example 2)
-
 Install the Redis bundle
 ************************
 
-In |CL|, find Redis in the :command:`redis-native` bundle.
+#. Log in as a user with administrative privilege.
 
-#. Open a terminal and login as an administrative user.
+#. Open a terminal.
```
Comment (mvincerx) [inline source/tutorials/redis.rst:30]: I recommend adding a swupd update here, before installing the bundle. 

I was testing on an older version (~32890), and for some reason, running `sudo systemctl enable --now redis`
just hung on creating a symlink to the proper `/etc` directory.

Let's eliminate this possibility by running a `sudo swupd update`

Comment (bktan8) [reply source/tutorials/redis.rst:30]: Did running "swupd update" fix your issue?

Comment (mvincerx) [reply source/tutorials/redis.rst:30]: Yes, it did. That's why I'm recommending it.

Comment (mvincerx) [reply source/tutorials/redis.rst:30]: Error occurred on GCE VM instance and on my physical NUC. It won't hurt to add this command.

**Comment by mvincerx** [comment]
LGTM.

**Comment by bktan8** [comment]
Thanks MV!!!

**Comment by mvincerx** [comment]
My pleasure. 😄

**Pull request merged**